### ISSUE TITLE
Numericality validation fails for currency with ',' delimiter.

### ISSUE THREAD

**Issue by achernik** [RubyMoney/money-rails#530]
This config leads to failure of numericality validation of `amount` attribute
```ruby
monetize :amount_cents
```
```
MoneyRails.configure do |config|
  config.default_currency = :rub

  config.amount_column = {
    prefix: '',
    postfix: '_cents',
    column_name: nil,
    type: :integer,
    present: true,
    null: false,
    default: 0
  }
  config.currency_column = {
    prefix: '',
    postfix: '_currency',
    column_name: nil,
    type: :string,
    present: true,
    null: false,
    default: 'RUB'
  }
end
I18n.default_locale = :ru
```

I was able to find the root cause:

In gem Money, new implementation of `#to_s` method uses `currency.decimal_mark` as separator, so `Money.new(0).to_s #=> '0,00'` ([here](https://github.com/RubyMoney/money/blob/f9482a1cb4d5f4901db111563222f8ef7588740a/lib/money/money.rb#L353-L359)).
However, in `money-rails`, [this method](https://github.com/RubyMoney/money-rails/blob/master/lib/money-rails/active_model/validator.rb#L63-L66) uses I18n, if enabled and available, so the decimal mark is not nesessarily the same. In my case it returns `'.'`, [normalize_raw_value!](https://github.com/RubyMoney/money-rails/blob/master/lib/money-rails/active_model/validator.rb#L118-L127) doesn't replace the separator, and this results in `super(...)` adding errors.

**Comment by achernik** [comment]
sorry, I accidentally hit the post button. I'll update issue description

**Comment by achernik** [comment]
fixed the description

**Comment by antstorm** [comment]
@achernik thanks a lot for digging into this, really helpful. I'm going to restore the i18n reliance in `#to_s` for now (https://github.com/RubyMoney/money/pull/771), but will figure out a proper solution to this in the next major release of `money`.

**Comment by antstorm** [comment]
I've pushed a new version with the fix — 6.11.3. I'm closing this issue for now, but feel free to re-open if it didn't work for you or you see any other regressions. Thanks again for reporting this.

**Comment by achernik** [comment]
This patch fixed the problem, thanks!
